conda_build/create_test.py

```diff
@@ -87,6 +88,28 @@ def create_shell_files(m):
                     f.write("if errorlevel 1 exit 1\n")
                 has_tests = True
 
+    libs = ensure_list(m.get_value('test/check_shared_libs_load', []))
+    if libs:
+        if sys.platform.startswith('win'):
+            lib_path = os.path.join(m.config.test_prefix, 'Library', 'bin')
+        else:
+            lib_path = os.path.join(m.config.test_prefix, 'lib')
+
+        with open(join(m.config.test_dir, name), 'a') as f:
+            f.write('\n\n')
+            for lib in libs:
+                lib_loc = os.path.join(lib_path, lib + get_shlib_ext())
+                # test existence
+                if sys.platform.startswith('win'):
+                    f.write('if not exist ' + lib_loc + ' exit 1\n')
+                else:
+                    f.write('test -f ' + lib_loc + '\n')
+                # test loading
+                f.write('python -c "import ctypes; ctypes.cdll[\'' + lib_loc + '\']"\n')
```

Comment:
@mingwandroid This is why we need `python` in `test/require` (and to make sure it uses that one), right? Then, assuming the library handles the VS features properly, it'll be loading with the right VS.

Using a compiler would mean installing the compiler in the test env, which then might end up bringing in libraries you accidentally linked against but didn't include in the test env...which would have masked exactly the problem that https://github.com/conda-forge/arpack-feedstock/pull/7 found.